src/ffi/pyerrors.rs

```diff
@@ -281,9 +282,11 @@ extern "C" {
     #[cfg_attr(PyPy, link_name = "PyPyExc_TimeoutError")]
     pub static mut PyExc_TimeoutError: *mut PyObject;
 
+    #[cfg(not(all(windows, PyPy)))]
     pub static mut PyExc_EnvironmentError: *mut PyObject;
+    #[cfg(not(all(windows, PyPy)))]
     pub static mut PyExc_IOError: *mut PyObject;
```

Comment:
Since `IOError` is just a alias of `OSError` on Python 3, I think we can expose `PyExc_IOError` as `PyPyExc_OSError`?

Use case: https://github.com/milesgranger/pyrus-cramjam/runs/2227542030?check_suite_focus=true

```
Installing collected packages: cramjam
Successfully installed cramjam-2.2.0
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: /opt/hostedtoolcache/PyPy/3.6.12/x64/site-packages/cramjam.pypy36-pp73-x86_64-linux-gnu.so: undefined symbol: PyExc_IOError
```

https://github.com/milesgranger/pyrus-cramjam/pull/48/commits/f60849c4f2f9992e2b7525b4479887be8808c22b#diff-76866598ce8fd16261a27ac58a84b2825e6e77fc37c163a6afa60f0f4477e569R343

Reply:
Hmm, but does it seem that PyPy takes a different strategy for that error?
https://foss.heptapod.net/pypy/pypy/-/blob/branch/default/pypy/module/exceptions/moduledef.py#L21

Reply:
I'm not sure, but it seems to be the same on the Python side on my local PyPy installation.

```
Python 3.7.10 (7035471788c8, Mar 19 2021, 02:02:14)
[PyPy 7.3.5-alpha0 with GCC Apple LLVM 12.0.0 (clang-1200.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
And now for something completely different: ``finally, mercurial migration is
happening!''
>>>> IOError
<class 'OSError'>
>>>> OSError
<class 'OSError'>
>>>> IOError is OSError
True
```

Reply:
Actually it is the same on pypy 3.6 and 3.7 branch https://foss.heptapod.net/pypy/pypy/-/blob/branch/py3.7/pypy/module/exceptions/moduledef.py#L31

Reply:
Ah, I got it, thanks.